sync.sh

```diff
@@ -2,9 +2,80 @@
 
 set -e
 
-# Read input arguments
+# Define Help function
+
+Help()
+{
+   # Display Help
+   echo "Tool to fill in belgian TST file from trade republic monthly report."
+   echo "This tool also creates a draft mail with the generated document and a qr code to pay the tax."
+   echo
+   echo "Syntax: sh sync.sh [-h|s|p|c] input_folder [output_folder]"
```

Comment:
Should I remove the default value of the input_folder argument and make it a mandatory argument ? Or do I mark it as optional here as well ?

Reply:
I guess we have to, we can't have both optional, we wouldn't know which one is being provided otherwise